Dockerfile

```
@@ -34,7 +34,12 @@ RUN npx playwright install chromium && \
 
 # Build application
 COPY . .
+# Build network interceptor bundle (must be before TypeScript compilation)
+RUN npm run build:bundle
+# Compile TypeScript
 RUN npm run build
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)